backends/tc/runtime/cleanup

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+RUNTIME_DIR=$1
+EXTRACT_DIR="$SCRIPT_DIR/extracted_deb"
+
+rm -rf $RUNTIME_DIR/iproute2-p4tc-pub
+rm -rf $EXTRACT_DIR
+rm -f $RUNTIME_DIR/id_rsa
```

Comment:
Seems like a dangerous script. Maybe add some checks to make sure that you are actually in the right folder?

Reply:
Sounds good! I think any of these are helpful

Reply:
I added a comment asking users not to run this manually, edited the script so that the `RUNTIME_DIR` is determined by the script itself (without user interference) and added a check to see whether `setup` and `cleanup` are in the same directory before running

Reply:
Great!